Index: certificate-app-frontend/App.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import { StatusBar } from 'expo-status-bar';\r\nimport React from 'react';\r\nimport { StyleSheet, Text, View } from 'react-native';\r\n\r\nexport default function App() {\r\n  return (\r\n    <View style={styles.container}>\r\n      <Text>Hello World</Text>\r\n      <StatusBar style=\"auto\" />\r\n    </View>\r\n  );\r\n}\r\n\r\nconst styles = StyleSheet.create({\r\n  container: {\r\n    flex: 1,\r\n    backgroundColor: '#fff',\r\n    alignItems: 'center',\r\n    justifyContent: 'center',\r\n  },\r\n});\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- certificate-app-frontend/App.js	(revision 5e70c1ebd5d3cc6bffc4a8a18ded95634445ddad)
+++ certificate-app-frontend/App.js	(date 1621779323374)
@@ -1,21 +1,72 @@
-import { StatusBar } from 'expo-status-bar';
+import 'react-native-gesture-handler';
+import { Camera } from 'expo-camera';
 import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import { render } from 'react-dom';
+import { View, Text, StyleSheet, TouchableOpacity, Button } from 'react-native';
+import { OpenCamera } from './classes/Camera.js';
+import { NavigationContainer } from '@react-navigation/native';
+import { createStackNavigator } from '@react-navigation/stack';
 
-export default function App() {
+function HomeScreen({ navigation }) {
   return (
     <View style={styles.container}>
-      <Text>Hello World</Text>
-      <StatusBar style="auto" />
+      <View style={styles.otherContainer}>
+        <Button
+        title="Camera"
+        onPress={() => navigation.navigate('Camera')
+        }>
+        </Button>
+      </View>
     </View>
-  );
+  )
+}
+
+const Stack = createStackNavigator();
+
+export default function App() {
+  return (
+    <NavigationContainer>
+      <Stack.Navigator initialRouteName="Home">
+        <Stack.Screen name="Home" component={HomeScreen} options={{ title: 'certificate-app', headerTitleStyle: { alignSelf: 'center' }}}/>
+        <Stack.Screen name="Camera" component={OpenCamera} />
+      </Stack.Navigator>
+    </NavigationContainer>
+  )
 }
 
-const styles = StyleSheet.create({
+var styles = StyleSheet.create({
+
   container: {
     flex: 1,
-    backgroundColor: '#fff',
+    flexDirection: 'column',
+    backgroundColor: 'white',
+    justifyContent: 'center',
     alignItems: 'center',
+  },
+  topBox: {
+    flex: 1,
+    flexDirection: 'row',
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  headline: {
+    textAlign: 'center',
+    fontWeight: 'bold',
+    fontSize: 24,
+    marginTop: 0,
+    width: 200,
+  },
+  marginTop: {
+    width: 200,
+    height: 80,
+  },
+  otherContainer: {
+    flex: 4,
     justifyContent: 'center',
+    alignItems: 'center',
+    backgroundColor: 'green',
   },
-});
+  button: {
+
+  },
+});
\ No newline at end of file
